Makefile

```diff
@@ -192,12 +192,8 @@ download-hugo: ## Download hugo software
 
 .PHONY: dev-docs
 dev-docs: download-hugo ## preview live your docs with hugo
-	@$(HUGO_BIN) server -s docs/ &
-	if type -p xdg-open 2>/dev/null >/dev/null; then \
-		xdg-open http://localhost:1313; \
-	elif type -p open 2>/dev/null >/dev/null; then \
-		open http://localhost:1313; \
-	fi
+	@echo "Starting hugo development server"
+	./hack/start-hugo-dev-server.sh $(HUGO_BIN) server -s docs/
```

Comment:
I tried to do this natively in in the Make file, but apparently make evaluates every line before executing any line, so something like `(xdg-open http://$(shell grep localhost $(HUGO_FIFO)))&` will evaluate to `(xdg-open http://)&` before the FIFO is even created by `mkfifo`. Even if you make the fifo inside a dependent target, Make will still evaluate the expansion first by waiting for the `grep` to return before executing the `hugo server` command